### PR TITLE
make Docker instructions for Development easier to discover and skim

### PR DESCRIPTION
**Pull request by Lotterleben**
I had a hard time figuring out what I did 8 months ago to get my Engelsystem dev system up & running. This PR attempts to make the necessary information easier to find and adds a "Troubleshooting" section that would've helped me and may just help someone else in the future.

### CODE REVIEW

**Review by xuwhite — CHANGES_REQUESTED** [review]
Thanks for the contribution. I have some small suggestions.

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -215,3 +217,11 @@ For some reason *PhpStorm* is unable to detect the server name.
 But without a server name it's impossible to set up path mappings.
 Because of that the docker setup sets the server name *engelsystem*.
 To get Xdebug working you have to create a server with the name *engelsystem* manually.
+
+## Troubleshooting
+
+### General tip
+If unspecific issues appear try using Docker version >= 20.10.14.
```
[the remark this answers is not in the document]
Comment (Lotterleben) [reply DEVELOPMENT.md:224]: This sentence is from your old docs, so I don't know which unspecific issues are referred to either 😅

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -215,3 +217,11 @@ For some reason *PhpStorm* is unable to detect the server name.
...
+
+### `service "es_workspace" is not running`
+make sure you're running your docker commends from `docker/dev`, not from `docker`
```
[the remark this answers is not in the document]
Comment (Lotterleben) [reply DEVELOPMENT.md]: fixed manually

**Comment by Lotterleben** [comment]
Addressed comments & squashed

File: DEVELOPMENT.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (xuwhite) [reply DEVELOPMENT.md:224]: True

**Pull request merged**